connect/resources/usage_automation.py

```diff
@@ -41,13 +44,23 @@ def filters(self, status='listed', **kwargs):
 
     def dispatch(self, request):
         # type: (UsageListing) -> str
+        handlers = [copy.copy(hdlr) for hdlr in global_logger.handlers]
+        log_level = global_logger.level
+        self.__class__.logger.propagate = False
+        self.__class__.logger.setLevel(log_level)
+        [self.__class__.logger.addHandler(hdlr) for hdlr in handlers]
+        base = " %(levelname)-6s; %(asctime)s; %(name)-6s; %(module)s:%(funcName)s:line" \
+               "-%(lineno)d: %(message)s"
+        sformat = request.id + "  " + request.contract.marketplace.id + base
+        [handler.setFormatter(logging.Formatter(sformat, "%I:%M:%S"))
+         for handler in self.__class__.logger.handlers]
 
```

Comment:
Mb use the default setting for logger
Right now we use thrice this block. Only field `sformat`  different  
```
handlers = [copy.copy(hdlr) for hdlr in global_logger.handlers]
        log_level = global_logger.level
        self.__class__.logger.propagate = False
        self.__class__.logger.setLevel(log_level)
        [self.__class__.logger.addHandler(hdlr) for hdlr in handlers]
        base = " %(levelname)-6s; %(asctime)s; %(name)-6s; %(module)s:%(funcName)s:line" \
               "-%(lineno)d: %(message)s"
        sformat = request.id + "  " + request.contract.marketplace.id + base
        [handler.setFormatter(logging.Formatter(sformat, "%I:%M:%S"))
         for handler in self.__class__.logger.handlers]
```

Reply:
Javi what you think about that suggestion?

Reply:
Hasn't that been fixed in the commits made afterwards?